scripts/compile-explorer

```diff
@@ -48,30 +48,40 @@ class CompileExplorer:
         if verbose:
             logger.setLevel(logging.NOTSET)
 
+    def _bid_boards_into_file(self, count, results_file):
+        bidder = Bidder()
+        results_file.write('[')
+        written = 0
+        try:
+            for _ in xrange(count):
+                # This whole dance is to avoid ^C adding a trailing comma.
+                result = self._bid_board(Board.random(), bidder)
+                if written != 0:
+                    results_file.write(',')
+                results_file.write('\n')
```

Comment:
there's a race here if the ^C comes in after the comma is put in the buffer. :-P lol